test/Abp.Zero.SampleApp.Tests/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -64,7 +64,8 @@ public void Should_Write_History_For_Tracked_Entities_Create()
                 entityChange.ChangeType.ShouldBe(EntityChangeType.Created);
                 entityChange.EntityId.ShouldBe(blog2Id.ToJsonString());
                 entityChange.EntityTypeFullName.ShouldBe(typeof(Blog).FullName);
-                entityChange.PropertyChanges.Count.ShouldBe(4);  // Blog.Name, Blog.Url, Blog.Category Blog.More
+                //Blog.Url, Blog.Category Blog.More, Blog.CreationTime, Blog.CreatorUserId, Blog.LastModificationTime, Blog.LastModifierUserId
+                entityChange.PropertyChanges.Count.ShouldBe(7);
```

Comment:
Also, we should create a separate audited entity to test here so that we can differentiate normal entity with audited attribute and audited entity